packages/cli/test/integration/passwordReset.endpoints.test.ts

```diff
@@ -76,16 +77,20 @@ test('POST /forgot-password should send password reset email', async () => {
 	config.set('userManagement.emails.smtp.auth.user', user);
 	config.set('userManagement.emails.smtp.auth.pass', pass);
 
-	const response = await authlessAgent
-		.post('/forgot-password')
-		.send({ email: INITIAL_TEST_USER.email });
+	await Promise.all(
+		[{ email: INITIAL_TEST_USER.email }, { email: member.email.toUpperCase() }].map(
```

Comment:
We should avoid using the shared `INITIAL_TEST_USER` in tests altogether. #3066 removes it.

Reply:
Nice with `Promise.all()`. We should remember to do this elsewhere later on.